scripts/cdn_publish.sh

```diff
@@ -1,26 +1,38 @@
 #!/bin/bash
 cdnGravityDir="gravity-ui-web"
 cdnDir="cdn-dist"
-allVersions=`git tag --list --sort=-committerdate | grep -E '^(gravity-ui-web-)?v[0-9]*.[0-9]*.[0-9]*'`
 
-# First tag in the list will be the latest one, but may be prefixed (e.g. "gravity-ui-web-v1.2.3")
-version=`echo $allVersions | cut -d" " -f1`
-# be sure we get only the version part and discard any prefix
-version=`expr $version : '.*\(v[0-9].*\)'`
+if [ ! -z "$TRAVIS_COMMIT_RANGE"]
+then
+  # TRAVIS_COMMIT_RANGE has a bug that includes commits outside the PR, so fix with known workaround that is future safe
+  cdnChanges=`git diff --pretty="format:" --name-only ${TRAVIS_COMMIT_RANGE/.../..} | grep -E '^packages/gravity-ui-web/src*'`
```

Comment:
Also according to this: https://github.com/travis-ci/travis-ci/issues/4596 the bug is fixed and merged. So we could probably just use `$TRAVIS_COMMIT_RANGE` instead of `${TRAVIS_COMMIT_RANGE/.../..}`

Reply:
I don't see any bug fix, only a closed issue.  Folks appear to continue to work around it.

Reply:
Looks like the Stale Bot closed it without resolution

Reply:
You're right. I was looking at this with (another) Dan and we saw some merged changes which did exactly this but looking again they're in a different package so it's not a fix - it's a work around that someone else is using 🤦‍♂